plaid/requester.py

```diff
@@ -34,8 +38,18 @@ def _requests_http_request(url, method, data, timeout=DEFAULT_TIMEOUT):
         )
 
 
-def http_request(url, method=None, data=None, timeout=DEFAULT_TIMEOUT):
-    response = _requests_http_request(url, method, data or {}, timeout)
+def http_request(
```

Comment:
let's keep the argument order consistent between `http_request` and `_requests_http_request` (the order of `headers` and `timeout` is reversed between the two).

Reply:
Great catch! You're right about the major release, my mistake.